setup.py

```diff
@@ -178,7 +178,12 @@ def detect_libb2(prefixes):
 
 
 with open('README.rst', 'r') as fd:
-    long_description = fd.read()
+    # remove badges
+    long_description = re.compile(r'^.. start-badges.*^.. end-badges', re.M | re.S).sub('', fd.read())
+    # remove |substitutions|
+    long_description = re.compile(r'\|\w+\|').sub('', long_description)
```

Comment:
I don't think we just want to kill them all (we would need to substitute them).

But as it currently is only |screencast|, maybe just specifically kill that?